src/libraries/System.Private.CoreLib/src/System/Number.NumberToFloatingPointBits.cs

```diff
@@ -14,23 +17,40 @@ internal unsafe partial class Number
                 exponentBits: 11,
                 maxBinaryExponent: 1023,
                 exponentBias: 1023,
-                infinityBits: 0x7FF00000_00000000
+                infinityBits: 0x7FF00000_00000000,
+                minDecimalExponent: -342,
```

Comment:
I'm not a huge fan of this name. My expectation when reading `Double.MinDecimalExponent` is that it's `-324` (the exponent of the smallest subnormal value). 

It might be better for this to explicitly be `minFastFloatDecimalExponent` or similar, just to ensure readers are well aware its not the "standard" decimal exponent.

Reply:
It might be worth putting `fastFloat` in the names of all the values here that are specific to `csFastFloat` and not more "general" to the IEEE 754 spec.